lib/node_modules/@stdlib/stats/base/dmskrange/README.md

```diff
@@ -184,6 +176,123 @@ console.log( v );
 
 <!-- /.examples -->
 
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dmskrange.h"
+```
+
+#### stdlib_strided_dmskrange( N, \*X, strideX, Mask, strideMask )
+
+Computes the [range][range] of a double-precision floating-point strided array `x` according to a `mask`.
```

Comment:
```suggestion
Computes the [range][range] of a double-precision floating-point strided array according to a mask.
```

In this case, the text variables don't match the signature in the line above. We can either use `X` and `Mask` or just remove. I am fine removing here. The main thing is that we don't want to be inconsistent in the immediate vicinity of the signature (e.g., `X` vs `x`, etc).

Reply:
noted, taking care of this everywhere else